chai_lab/chai1.py

```diff
@@ -693,15 +701,17 @@ def avg_per_token_1d(x):
                 for c in feature_context.chains
             },
         )
-        output_paths.append(cif_out_path)
+        cif_paths.append(cif_out_path)
 
-        scores_basename = f"scores.model_idx_{idx}.npz"
-        scores_out_path = output_dir / scores_basename
+        scores_out_path = output_dir.joinpath(f"scores.model_idx_{idx}.npz")
 
-        scores = get_scores(ranking_outputs)
-        np.savez(
-            scores_out_path,
-            **scores,
-        )
+        np.savez(scores_out_path, **get_scores(ranking_outputs))
```

Comment:
Seems like we're saving this but not returning a path to it; should we return the path or skip saving it altogether?

Reply:
valuable part of info is duplicated in output, sounds fine to skip